src/Models/Actions/TemplatedEmail.cs

```diff
@@ -17,7 +17,6 @@ public TemplatedEmail()
         public override Task ProcessTemplatedEmail(IActionHelper actionHelper, ITemplatedEmailProvider templatedEmailProvider, Dictionary<string, dynamic> personalisation, FormAnswers formAnswers)
         {
             var emailAddressList = actionHelper.GetEmailToAddresses(this, formAnswers).Split(',').ToList();
-            personalisation.Add("reference", formAnswers.CaseReference);
```

Comment:
As CaseReference is not a question it won't be present in the formAnswers.AllAnswers but we still need to be able to have it as a personalisation somehow